src/contracts/test/TestEmptyDiscountRateStrategy.t.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import './TestGhoBase.t.sol';
+
+import {EmptyDiscountRateStrategy} from '../facilitators/aave/mocks/EmptyDiscountRateStrategy.sol';
+
+contract TestEmptyDiscountRateStrategy is TestGhoBase {
+  EmptyDiscountRateStrategy emptyStrategy;
+
+  function setUp() public {
+    emptyStrategy = new EmptyDiscountRateStrategy();
+  }
+
+  function testFuzzConstructor(uint256 debtBalance, uint256 discountTokenBalance) public {
```

Comment:
I think this should be named differently right? Maybe `testFuzzRateAlwaysZero` or something